src/Azure.Functions.Cli/Common/Utilities.cs

```diff
@@ -265,29 +247,39 @@ internal static bool DefaultLoggingFilter(string category, LogLevel actualLevel,
         /// </summary>
         /// <param name="actualLevel"></param>
         /// <returns></returns>
-        internal static bool UserLoggingFilter(LogLevel actualLevel)
+        internal static bool UserLoggingFilter(LogLevel actualLevel, LogLevel minLogLevel)
         {
             if (actualLevel == LogLevel.None)
             {
                 return false;
             }
-            return actualLevel >= LogLevel.Trace;
+            return actualLevel >= minLogLevel;
         }
 
         internal static bool SystemLoggingFilter(string category, LogLevel actualLevel, LogLevel minLevel)
         {
             return actualLevel >= minLevel && IsSystemLogCategory(category);
         }
 
+        internal static bool JobHostSystemLoggingFilter(string category, LogLevel actualLevel, LogLevel minLevel)
+        {
+            if(IsSystemLogCategory(category))
+            {
+                return true;
+            }
+            return actualLevel >= minLevel;
+        }
+
         internal static bool IsSystemLogCategory(string category)
         {
-            return SystemCategoryPrefixes.Where(p => category.StartsWith(p)).Any();
+            return ScriptConstants.SystemLogCategoryPrefixes.Where(p => category.StartsWith(p)).Any();
         }
 
         internal static IConfigurationRoot BuildHostJsonConfigutation(ScriptApplicationHostOptions hostOptions)
         {
             IConfigurationBuilder builder = new ConfigurationBuilder();
             builder.Add(new HostJsonFileConfigurationSource(hostOptions, SystemEnvironment.Instance, loggerFactory: NullLoggerFactory.Instance, metricsLogger: new MetricsLogger()));
+            builder.Add(new ScriptEnvironmentVariablesConfigurationSource());
```

Comment:
This doesn't seem like the right place to add another configuration source

Reply:
not needed anymore. Thanks!

Reply:
Need to figure out what is the right place to add this configuration source. Without this overriding via local.setting.json is not working.

Reply:
Using `LoggerFilterOptions` at webhost level fixed this.